src/pages/groupComparison/UpSet.tsx

```diff
@@ -208,11 +199,15 @@ export default class UpSet extends React.Component<IUpSetrProps, {}> {
         return index * (this.barWidth() + this.barSeparation()); // half box + separation + half box
     }
 
+    @computed get uidToGroup() {
+        return _.keyBy(this.props.groups, group=>group.uid)
```

Comment:
why do you want to remake uidToGroup here instead of reusing the one thats passed in?